views/applications/one.html

```diff
@@ -25,15 +26,16 @@ <h1 class="introduction--h1">{{config.actions.edit|capitalize}}</h1>
   {% if app.client.secret %}
     <dl class="base--dl">
       <dt class="base--dt">Client Secret</dt>
-      <dd class="base--dd">{{app.client.secret}} (<a href="#TBD">Create new Client Secret</a>)</dd>
+      <dd class="base--dd">{{app.client.secret}} (<a href="/applications/secret">Create new Client Secret</a>)</dd>
     </dl>
   {% endif %}
 
   {% if app.endpoints %}
     <h2>Endpoints</h2>
 
-    {% if app.endpoints.live %}
-      <h3 id="live">Live</h3>
+    {% if app.endpoints.live | length %}
+      <h3>Live</h3>
+      <a name="live"></a>
```

Comment:
Why were these changed from `id` to `name`? `id` is semantically correct and `name` [has been deprecated](https://www.w3.org/TR/html5/obsolete.html) on anchor tags. Please return this to having `id` on the `h3`